clients/include/rocblas.hpp

```diff
@@ -39,6 +39,18 @@
          const void *x, rocblas_int incx,
          void *y, rocblas_int incy);
 
+     rocblas_status
+     rocblas_set_matrix( rocblas_int rows, rocblas_int cols, 
+         rocblas_int elem_size,
+         const void *a, rocblas_int lda,
+         void *b, rocblas_int ldb);
+
+     rocblas_status
+     rocblas_get_matrix( rocblas_int rows, rocblas_int cols, 
+         rocblas_int elem_size,
+         const void *a, rocblas_int lda, 
+         void *b, rocblas_int ldb);
+
```

Comment:
In the C++ header, these could be templated.  You could call the routine like:
`rocblas_set_matrix<double>(rocblas_int rows, ... );`
and this removes the need for `rocblas_int elem_size` parameter.